airflow/migrations/versions/8d48763f6d53_add_unique_constraint_to_conn_id.py

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""add unique constraint to conn_id
+
+Revision ID: 8d48763f6d53
+Revises: 8f966b9c467a
+Create Date: 2020-05-03 16:55:01.834231
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+from airflow.models.base import ID_LEN
```

Comment:
This value can change in future.

So we should hard code it in the migration itself, instead of importing